### PR TITLE
Changed the deferred examples to mimic jQuery's deferred

### PR DESCRIPTION
**Pull request by softwarespot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1086,14 +1086,16 @@ A promise represents the eventual result of an asynchronous operation. jQuery ha
...
+    const defer = new $.Deferred((deferred) => {
+      setTimeout(() => {
+        if(true) {
+          deferred.resolve('some_value_computed_asynchronously');
+        } else {
+          deferred.reject('failed');
+        }
+      }, 1000);
+    });
+
```
Comment (camsong) [inline README.md]: I prefer the previous style of using `$.Deferred`, as the example in this jQuery document http://api.jquery.com/deferred.promise/

**Comment by softwarespot** [comment]
Ah okay, I was comparing with how you do it with Promise. Okay, so I will update the PR and squash the commits.

**Comment by softwarespot** [comment]
Updated the PR with the changes mentioned beforehand.

**Comment by camsong** [comment]
Great and thanks 👍

**Comment by camsong** [comment]
@softwarespot Thanks so much for your contributions on this project, I'm really impressed and would like to invite you as one of our collaborators, let's make this project better and help more developers 👏  💕

**Comment by softwarespot** [comment]
No problem. You don't have to make me a collaborator, but it would be much appreciated none the less =)

**Pull request merged**